src/printer/layers.js

```diff
@@ -465,6 +470,152 @@ function createLayerWFS(jobId, layerSpec, rootFrameState) {
   return progress$;
 }
 
+/**
+ * @param {number} jobId
+ * @param {import('../main/index.js').BingMapsLayer} layerSpec
+ * @param {import('ol/Map').FrameState} rootFrameState
+ * @return Promise<Observable<LayerPrintStatus>>
```

Comment:
I think this PR is almost good to go, I would just like to avoid this kind of pattern (mixing promises and observables). Obviously here we have to return an observable like the other layers, so how about wrapping the first part of the function in a separate private function like `createLayerBingMapsInternal` which would return a promise resolving to the OL layer, and then integrate that in the final observable returned by `createLayerBingMaps`?